.github/workflows/release.yml

```diff
@@ -66,8 +66,7 @@ jobs:
         run: make test
   update-api-specs:
     runs-on: ubuntu-latest
-    needs: [ token-validation ]
-#     needs: [test]
+    needs: [ test ]
```

Comment:
@shamantraghav  -Addressed merge issue, could you please help here if this should be the expected behaviour? 

Kept just test as dependency, since test already depends upon token-validation.

Reply:
This should be fine now as tests are enabed